CMakeLists.txt

```
@@ -318,6 +318,12 @@ if(LINUX)
   set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-as-needed")
 endif()
 
+# This is a reordering fix for relocation marker overflow for CUDA 11.1
+# Inspired by https://www.internalfb.com/intern/codesearch/path/fbsource/tools/build_defs/fbcode_macros/build_defs/lib/cpp_common.bzl?lines=882
```

not sure if you should use internalfb link in here. i think just replace it by the actual comment should suffice 
```suggestion
    # Reorder nv_fatbin after the bss section to avoid overflow, this only works for lld linker
```

---

oh true